code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -184,3 +184,31 @@
 			message = replacetextEx(message,regex(capitalize(key),"g"), "[capitalize(value)]")
 			message = replacetextEx(message,regex(key,"g"), "[value]")
 	speech_args[SPEECH_MESSAGE] = trim(message)
+
+/obj/item/clothing/mask/Madvilliansmask
+	name = "the MadVillian's mask"
+	desc = "A metal mask formerly owned by the Madvillian."
+	icon_state = "madvilliansmask"
+	inhand_icon_state = "madvilliansmask"
+	w_class = WEIGHT_CLASS_SMALL
+	modifies_speech = TRUE
+
+/obj/item/clothing/mask/Madvilliansmask/dropped(mob/M)
+	. = ..()
+	UnregisterSignal(M, COMSIG_MOB_SAY)
```

Comment:
why is this here? you never register a signal

Reply:
It unregisters the signal COSMIG_MOB_SAY puts out that modifies speech, so when you take it off it no longer modifies words to DOOMSPEAK

Reply:
what? you never register for this signal so why unregister for it.